app/javascript/dashboard/store/modules/automations.js

```diff
@@ -55,6 +55,17 @@ export const actions = {
       commit(types.SET_AUTOMATION_UI_FLAG, { isUpdating: false });
     }
   },
+  toggle: async ({ commit }, { id, ...updateObj }) => {
```

Comment:
Add spec for `toggle` action

Reply:
@muhsin-k removed this entire method, as we fixed the update method.